config/default.yml

```diff
@@ -442,7 +442,7 @@ RSpec/ExpectOutput:
 
 RSpec/FilePath:
   Description: Checks that spec file paths are consistent and well-formed.
-  Enabled: true
+  Enabled: false
```

Comment:
Question: The changelog reads that `RSpec/FilePath` is "enabled by default" but it seems it isn't anymore.

Currently, users are forced to replace `RSpec/FilePath` with their alternatives when upgrading a minor release from 2.23.0 to 2.24.0. Is this expected or do we consider this a breaking change? :thinking: 

Refs https://gitlab.com/gitlab-org/ruby/gems/gitlab-styles/-/merge_requests/195 

/cc @ydah @bquorning

Reply:
I can open an issue to continue the discussion if needed :heart:

Reply:
Yes, please open an issue. I am not sure what is the preferred course of action – should we add the old cop back?

Reply:
@ydah @bquorning I've moved the discussion to https://github.com/rubocop/rubocop-rspec/issues/1717.